draft-ietf-scitt-architecture.md

```diff
@@ -86,7 +86,16 @@ normative:
 informative:
 
   I-D.draft-ietf-cose-merkle-tree-proofs: COMETRE
-
+  I-D.draft-ietf-rats-eat: draft-ietf-rats-eat
+  NIST.SP.1800-19:
```

Comment:
Can someone explain to me what an NCCoE implementation guide regarding "Security Practice Guide for VMware Hybrid Cloud Infrastructure as a Service (IaaS) Environments" has to do with SCITT?

```suggestion
```

Reply:
more detailed answer: because NIST defined "assertion" and "attestation" which are terms that need disambiguation in the context of SCITT.